std/experimental/color/conv.d

```diff
@@ -0,0 +1,523 @@
+// Written in the D programming language.
+
+/**
+    This module implements various _color type conversions.
+
+    Authors:    Manu Evans
+    Copyright:  Copyright (c) 2015, Manu Evans.
+    License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
+    Source:     $(PHOBOSSRC std/experimental/color/conv.d)
+*/
+module std.experimental.color.conv;
+
+import std.experimental.color;
+import std.experimental.color.rgb;
+import std.experimental.color.xyz;
+
+import std.traits: isNumeric, isIntegral, isFloatingPoint, isSigned, TemplateOf;
+import std.typetuple: TypeTuple;
+
+
+/**
+Convert between color types.
+*/
+To convertColor(To, From)(From color) if(isColor!To && isColor!From)
```

Comment:
functions you can view without scrolling are preferable.

```d
To convertColor(To, From)(From color) if(isColor!To && isColor!From && is(To == From))
```
would be the first

as I said, the static ifs inside the function are natural breakpoints

the else block must have the negated constraints of the other functions